src/session-pool.js

```diff
@@ -59,7 +59,7 @@ function SessionPool(database, options) {
   options = options || {};
 
   this.database = database;
-  this.maxIdle = options.maxIdle || 1;
+  this.maxIdle = options.maxIdle || options.min || 1;
```

Comment:
Is this a place where `Math.min` would help?

Reply:
Or `max`?

Reply:
```
> Math.min(undefined, 1)
<- NaN
```

So, no. :-)

Reply:
```js
this.maxIdle = options.maxIdle /*0*/ || options.min /*10*/ || 1
// this.maxIdle = 10
```

Is this scenario realistic and assignment correct?

Reply:
@stephenplusplus I just pushed a commit that checks to see if `options.maxIdle` is a number first.